install.ps1

```
@@ -0,0 +1,86 @@
+                    }
+                )
+            }
+        )
+    }
+    permissions = @{
+        allow = @("Skill")
+    }
+}
+
+$settings | ConvertTo-Json -Depth 10 | Set-Content $SettingsFile -Encoding UTF8
+Write-Host "✓ ~/.claude/settings.json 설정 완료" -ForegroundColor Green
```

PSScriptAnalyzer (1.24.0)

[warning] File 'install.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)